frontend/src/components/managePermissions/table/utils.ts

```diff
@@ -1,13 +1,12 @@
 import { hashRouter } from "@/router";
-import { routeItems } from "@/router/routeItem";
 import { selectedApplicationId } from "@/store/ApplicationState";
 
 export const navigateToUserDetails = (userId: string) => {
     hashRouter.push({
-        name: routeItems.userDetails.name,
+        name: "UserDetails",
```

Comment:
unfortunately not since routeItems is an object of objects, my goal with the routeItems is to define everything in one place, this will simplify the process when we need to add a new path. I could think of something better to avoid this hard coding problem. Maybe i'll make a getter function for path names

Reply:
https://github.com/bcgov/nr-forests-access-management/pull/1627/commits/2caca8e08472b6fbb3dd1927ba03781c090d3628

got rid of them here

Reply:
No worry, this hardcode might be something can be ignored, I am just wondering if there is a better way to reuse them. Thanks.